src/app/lib/preventChangeOnNumberInputScroll.ts

```diff
@@ -0,0 +1,19 @@
+import { TextInputProps } from 'grommet/es6/components/TextInput'
+
+export const preventChangeOnNumberInputScroll = (): Pick<TextInputProps, 'onWheel'> => {
+  return {
+    onWheel: e => {
```

Comment:
https://github.com/oasisprotocol/oasis-wallet-web/pull/1964

I added `.preventDefault()` implementation with global listener. It behaves differently. Since focus is captured, you won't be able to scroll on top of the input[type="number"] field. Which makes it a bit awkward IMO. The current implementation will let you scroll the whole page while hovering over input[type="number"] field.